debian/postinst

```diff
@@ -13,6 +13,12 @@ case "$1" in
         # Create Symlink
         ln -s /usr/share/sonic-pi/app/gui/qt/sonic-pi /usr/bin/make-music
 
+        if `dpkg --compare-versions $2 lt 2.9-0`; then
+            # v2.9-0: New defaults for workspace size so remove old ones
+            find /home -path /home/*/.config/uk.ac.cam.cl/Sonic\ Pi.conf \
```

Comment:
I'm not certain but it definitely was on my image. Had you loaded up Sonic Pi before checking?